packages/wrangler/src/dev/local.tsx

```diff
@@ -67,7 +67,7 @@ function useLocalWorker({
 	logLevel,
 }: LocalProps) {
 	// TODO: pass vars via command line
-	const local = useRef<ReturnType<typeof spawn>>();
+	const local = useRef<ReturnType<typeof fork>>();
```

Comment:
Can you just use the ChildProcess type?

```suggestion
	const local = useRef<ChildProcess>();
```